packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -58,6 +64,9 @@ export default util.createRule<Options, MessageIds>({
     messages: {
       missingAccessibility:
         'Missing accessibility modifier on {{type}} {{name}}.',
+      addPublicAccessibility: "Add 'public' accessibility modifier",
+      addPrivateAccessibility: "Add 'private' accessibility modifier",
+      addProtectedAccessibility: "Add 'protected' accessibility modifier",
```

Comment:
See the other messages: we can reuse the same message ID by having a `data: { type: ... }` provided along with it.

```suggestion
      addExplicitAccessibility: "Add '{{ type }}' accessibility modifier",
```